ElectronClient/main-html.js

```diff
@@ -76,6 +76,7 @@ BaseItem.loadClass('Revision', Revision);
 
 Setting.setConstant('appId', 'net.cozic.joplin-desktop');
 Setting.setConstant('appType', 'desktop');
+Setting.setConstant('isDarkMode', bridge().systemTheme());
```

Comment:
`shouldUseDarkColors` is a dynamic value so why do you set it as constant? If the user changes the system theme, the app should update too.

Reply:
I tried `Setting.setValue('isDarkMode', bridge().systemTheme())` but that didn't work here so I set as a constant. 
I'll try again to make it dynamic